pkg/store/github/github.go

```diff
@@ -250,30 +252,48 @@ func (s *GitHubStore) ReloadProviderCache(ctx context.Context) error {
 
 		var versions []core.ProviderVersion
 		for _, release := range releases {
-			var platforms []core.Platform
 			version := strings.TrimPrefix(release.GetName(), "v")
 
+			if s.providerIgnoreCache[cacheKey(nameKey, version)] {
+				s.logger.Debug(fmt.Sprintf("Ignoring release [%s/%s], previously found to be not valid", nameKey, version))
```

Comment:
style: don't capitalise the first letter of the first word in log messages if it's only because it's the start of the sentence